### PR TITLE
fix(Scripts): Crashfix.

### PR DESCRIPTION
**Pull request by UltraNix**
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/instance_the_black_morass.cpp [hunk 1]
```diff
@@ -130,7 +130,7 @@ class instance_the_black_morass : public InstanceMapScript
                                     case NPC_INFINITE_EXECUTIONER_2:
                                     case NPC_INFINITE_VANQUISHER:
                                     case NPC_INFINITE_VANQUISHER_2:
-                                        creature->DespawnOrUnsummon();
+                                        creature->DespawnOrUnsummon(1);
```
Comment (Kitzunu) [inline src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/instance_the_black_morass.cpp:133]:
```suggestion
                                        creature->DespawnOrUnsummon(2s);
```

Comment (mpfans) [reply src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/instance_the_black_morass.cpp]: compilation Error

File: src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/instance_the_black_morass.cpp [hunk 2]
```diff
@@ -130,7 +130,7 @@ class instance_the_black_morass : public InstanceMapScript
                                     case NPC_INFINITE_EXECUTIONER_2:
                                     case NPC_INFINITE_VANQUISHER:
                                     case NPC_INFINITE_VANQUISHER_2:
-                                        creature->DespawnOrUnsummon();
+                                        creature->DespawnOrUnsummon(2s);
```
Comment (Kitzunu) [inline src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/instance_the_black_morass.cpp:133]:
```suggestion
                                        creature->DespawnOrUnsummon(1);
```

File: src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/instance_the_black_morass.cpp (hunk 1 above)
Comment (Kitzunu) [reply src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/instance_the_black_morass.cpp:133]: My memory was failing me, my bad

**Comment by Mitradis** [comment]
@Winfidonarleyan я точно помню, что было исправление квеста на приручение петомца ханта и там фигурировал этот момент, который как то мешал пету лока и пустое место в ковычках не просто так поставили. Если вернуть цифру чтото сломается в призыве или в квесте на ханта.

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**